config.json

```diff
@@ -0,0 +1,541 @@
+          "settings": {
+            "CloudDropAccountName": "default"
+          }
+        },
+        "verbose": {
+          "description": "Passes /flp:v=diag to the msbuild command or the value passed by the user.",
+          "settings": {
+            "MsBuildFileLogging": "/flp:v=diag;LogFile=sync.log"
+          }
+        },
+        "buildMajor": {
+          "description": "To download a specific group of product packages, specify build number. The value for -BuildMinor required.",
+          "settings": {
+            "BuildNumberMajor": "default"
+          }
+        },
+        "buildMinor": {
+          "description": "To download a specific group of product packages, specify build number. The value for -BuildMajor required.",
+          "settings": {
+            "BuildNumberMinor": "default"
+          }
+        }
+      },
+      "defaultValues": {
+        "toolName": "msbuild",
+        "settings": {
+          "MsBuildParameters": "default",
+          "maxcpucount": "default",
+          "MsBuildFileLogging": "/flp:v=detailed;LogFile=sync.log"
+        }
+      }
+    },
+    "publish-packages": {
+      "alias": {
+        "azureAccount": {
+          "description": "Account token to connect to Azure Blob storage.",
+          "settings": {
+            "CloudDropAccessToken": "default"
+          }
+        },
+        "azureToken": {
+          "description": "Account name to connect to Azure Blob storage.",
+          "settings": {
+            "CloudDropAccountName": "default"
+          }
+        },
+        "buildArch": {
+          "description": "Specifies architecture to publish, can be x64, x86, arm or arm64",
+          "settings": {
+            "__BuildArch": "default"
+          }
+        },
+        "buildType": {
+          "description": "Specifies configuration to publish, can be Release, Debug or Checked.",
+          "settings": {
+            "__BuildType": "default"
+          }
+        },
+        "buildOS": {
+          "description": "Specifies the OS to publish packages.",
+          "settings": {
+            "__BuildOS": "default"
+          }
+        }
+      },
+      "defaultValues": {
+        "toolName": "msbuild",
+        "settings": {
+          "Project": "./src/publish.proj",
+          "MsBuildParameters": "default",
+          "maxcpucount": "default",
+          "MsBuildFileLogging": "/flp:v=detailed;LogFile=publish-packages.log"
+        }
+      }
+    },
+    "build-packages": {
+      "alias": {
+        "buildArch": {
+          "description": "Specifies architecture to publish, can be x64, x86, arm or arm64",
+          "settings": {
+            "__BuildArch": "default"
+          }
+        },
+        "buildType": {
+          "description": "Specifies configuration to publish, can be Release, Debug or Checked.",
+          "settings": {
+            "__BuildType": "default"
+          }
+        },
+        "distroRid": {
```

Comment:
distroRid is better than osversion, since this provides the full runtime id, for ex, this value will be ubuntu.16.04-x64, where osversion will be ubuntu.16.04. And using buildOS here since that's the name of setting used through the coreclr build targets.